tests/java/InferBearTest.py

```diff
@@ -20,8 +22,38 @@ class InferBad {
 }
 """
 
+good_c_file = """
+#include <stdlib.h>
+
+void test() {
+  int *s = NULL;
+  if (s != NULL) {
+    *s = 42;
+  }
+}
+"""
+
+bad_c_file = """
+#include <stdlib.h>
+
+void test() {
+  int *s = NULL;
+  *s = 42;
+}
+"""
+
+
+InferBearJavaTest = verify_local_bear(
+    InferBear, valid_files=(good_java_file,), invalid_files=(bad_java_file,),
```

Comment:
Need some formatting here.

Reply:
Yes, keep all params in separate lines. valid_files on next line and invalid again on next line and so on.